### ISSUE TITLE
bundler install arguments "--binstubs" and "--without" being deprecated

### ISSUE THREAD

**Issue by mmuman** [sleede/fab-manager#362]
While experimenting with dockerless install I noticed `bundler` complains about the arguments used in the Dockerfile being deprecated, so I suppose these will need to be changed someday:

```
[DEPRECATED] The `--without` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set --local without 'development test doc'`, and stop using this flag
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and installing your bundle as
root will break this application for all non-root users on this machine.
[DEPRECATED] The --binstubs option will be removed in favor of `bundle binstubs --all`
```

**Comment by sylvainbx** [comment]
Hi @mmuman , thank for your report. We will improve that as soon as possible.

**Comment by mmuman** [comment]
I did this locally and it seemed to work:
```
bundle config set --local without 'development test doc'
bundle install
bundle binstubs --all
```
Not doing so actually generated errors further on in the install process.

**Comment by sylvainbx** [comment]
This should be fixed in 5.4.3
